d2-test-api/src/main/java/com/linkedin/d2/balancer/clients/TestClient.java

```diff
@@ -0,0 +1,153 @@
+/*
+   Copyright (c) 2020 LinkedIn Corp.
```

Comment:
From what I remember, the policy is to keep the original copyright date if this is a simple refactor of the code.
This is the year of first publication.